src/PaymentInformation.php

```diff
@@ -164,6 +164,21 @@ class PaymentInformation
      */
     protected $dateFormat = 'Y-m-d';
 
+    /**
+     * @var string
+     */
+    protected $serviceLevelCode = "SEPA";
+
+    /**
+     * @var string|null
+     */
+    protected $chargeBearer = "SLEV";
+
+    /**
+     * @var bool
+     */
+    protected $showCurrency = true;
```

Comment:
Se comment above for `&& $paymentInformation->showCurrency()`

Reply:
See comment above. This is for backwards compatibility.

Reply:
Now I am back on track, this is not the suitable place as this is about how to build the XML files, this should reside in the DOMBuilders and not in the data structure.